astropy/utils/data.py

```diff
@@ -120,6 +126,10 @@ def _is_url(string):
     return url.scheme.lower() in ['http', 'https', 'ftp', 'sftp', 'ssh', 'file']
```

Comment:
What motivates this particular subset of the IANA schemes? ( https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml ) I suppose we should use the list of schemes that `urllib.request` supports, though I don't see that list anywhere obvious.

Reply:
Do we want to support `data://` URLs? `urllib.request` does, and Jupyter notebooks (among other sources) use them for images.

Reply:
Behavior change is out of scope.